ivadomed/utils.py

```diff
@@ -61,6 +62,11 @@ def pred_to_nib(data_lst, z_lst, fname_ref, fname_out, slice_axis, debug=False,
             for arr in tmp_lst:
                 print("Shape element lst {}".format(arr.shape))
 
+        # single slice can be mistaken with channel we add an empty slice in the volume
+        if len(tmp_lst) == 1:
```

Comment:
1 channel should not be mistaken with 1 slice dimensions of the input image should be  (channel x height x width x depth). Our approach should be robust to 1 slice without conditions. As I mentioned on slack these lines shoul be removed :
 ```   
 # If only one channel then return 3D arr
    if arr_pred_ref_space.shape[0] == 1:
        arr_pred_ref_space = arr_pred_ref_space[0, :]
```

Reply:
Well depends on what you call height, width, and depth since we orient it to its original shape just before...I think it needs more extensive testing..